src/App.vue

```diff
@@ -67,9 +67,6 @@ import { EventBus } from './services/EventBus.js'
 import { leaveConversationSync } from './services/participantsService.js'
 import { signalingKill } from './utils/webrtc/index.js'
 
-// Styles
-import '@nextcloud/dialogs/dist/index.css'
```

Comment:
As per https://github.com/nextcloud/files_retention/pull/399#discussion_r1388088157 I misunderstood it and the file name was just wrong.